src/main/resources/db/changelog/changes/1-create-info-phoneNumber-table-changelog.sql

```diff
@@ -0,0 +1,15 @@
+--liquibase formatted sql
+
+--changeset noyan:1
+
+CREATE TABLE `phone_numbers` (
+     `id` bigint NOT NULL AUTO_INCREMENT,
+     `create_date` datetime(6) DEFAULT NULL,
```

Comment:
please change datetime types to timestamp.

Reply:
I just changed datetime types to timestamp.